ScheduleParser/ScheduleParser.cs

```diff
@@ -0,0 +1,223 @@
+                (row.GetCell(0).StringCellValue, row.GetCell(4).StringCellValue)
+            );
+        }
+
+        foreach (var studentWork in meeting.StudentWorks)
+        {
+            var student = studentWork.StudentName;
+            foreach (var pair in studentsAndConsultants.Where(pair => student == pair.Item1))
+            {
+                studentWork.Consultant = pair.Item2;
+            }
+        }
+    }
+    
+    /// <summary>
+    /// Gets the chair sheet from the table with themes related to this level of education.
+    /// </summary>
+    private ISheet GetChairSheet(string chair, string level)
+    {
+        var stream = GetStream(level switch
+        {
+            "бакалавры МОиАИС" => config.BachelorsMs,
+            "бакалавры ПИ" => config.BachelorsSe,
+            "магистры МОиАИС" => config.MastersMs,
+            "магистры ПИ" => config.MastersSe,
+            _ => throw new ArgumentException($"Неверный уровень образования: {level}")
+        });
+        var workbook = new XSSFWorkbook(stream);
+
+        return chair == "Информатика/ПА"
+            ? MergeSheets([workbook.GetSheet("Информатики"), workbook.GetSheet("ПА")])
+            : workbook.GetSheet(chair);
+    }
+    
+    /// <summary>
+    /// Merge sheets with same header.
+    /// </summary>
+    private ISheet MergeSheets(List<ISheet> sheets)
+    {
+        var newSheet = new XSSFWorkbook().CreateSheet();
+        var newRowIndex = 0;
+
+        var header = newSheet.CreateRow(newRowIndex);
+        foreach (var cell in sheets[0].GetRow(0))
+        {
+            header.CreateCell(cell.ColumnIndex).SetCellValue(cell.StringCellValue);
+        }
+
+        foreach (var sheet in sheets)
+        {
+            for (var i = 1; i < sheet.LastRowNum; i++)
+            {
+                var row = sheet.GetRow(i);
+                newRowIndex++;
+                var newRow = newSheet.CreateRow(newRowIndex);
+                foreach (var cell in row)
+                {
+                    newRow.CreateCell(cell.ColumnIndex).SetCellValue(cell.StringCellValue);
+                }
+            }
+        }
+
+        return newSheet;
+    }
+    
+    /// <summary>
+    /// Fetches information about the student work.
+    /// </summary>
+    /// <param name="cells">Cells with information about the student work.</param>
+    /// <returns>Instance of <see cref="StudentWork"/> class.</returns>
+    private StudentWork FetchStudentWork(List<ICell> cells)
+    {
+        var number = cells[0].NumericCellValue;
+        var studentName = cells[1].StringCellValue;
+        var theme = cells[2].StringCellValue;
+        var supervisor = cells[3].StringCellValue;
+        var reviewer = cells[4].StringCellValue;
+
+        return new StudentWork((int)number, studentName, theme, supervisor, "", reviewer);
+    }
+
+    private bool CellsAreEmpty(List<ICell> cells)
+    {
+        return cells.All(cell => cell.CellType == CellType.Blank);
+    }
```

Comment:
```suggestion
    private bool CellsAreEmpty(List<ICell> cells)
        => cells.All(cell => cell.CellType == CellType.Blank);
```
И ниже, для всех методов-однострочников. А ещё, если метод не использует состояние объекта, его надо сделать static-ом